### PR TITLE
Updating the register component. 3 Naming conventions to be updated. …

### PR DESCRIPTION
**Pull request by boneswvb**
Updating the register component. 3 Naming conventions to be updated. State and API done

### CODE REVIEW

**Review by olajohn-ajiboye — APPROVED** [review]
This looks good, can you please remove the `console.log`😊
One other suggestion is to also use just one changehandle to handle all form input. You can leave as it is because of time.

File: src/pages/SignUpPage.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+// User Signup
+// http://moboclinic.herokuapp.com/api/register
+
+// {
+// 	"first_name": "segun",
+// 	"last_name": "aka",
+// 	"email": "segun@test.com",
+// 	"phone": "080123456887",
+// 	"password": "testing"
+// }
```
Comment (olajohn-ajiboye) [inline src/pages/SignUpPage.js]:
```suggestion

```

File: src/pages/SignUpPage.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            console.error(err.message)
+        }
+    }
+
+    console.log('first_name', first_name)
+    console.log('last_name', last_name)
+    console.log('email', email)
+    console.log('phone', phone)
+    console.log('password', password)
+    console.log('password_confirmation', password_confirmation)
```
Comment (olajohn-ajiboye) [inline src/pages/SignUpPage.js]:
```suggestion

```

File: src/pages/SignUpPage.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        e.preventDefault()
+        try {
+            const body = { first_name, last_name, email, phone, password }
+            const response = fetch('http://moboclinic.herokuapp.com/api/register', {
+                method: 'POST',
+                headers: { 'Content-Type': 'application/json' },
+                body: JSON.stringify(body),
+            })
+            window.location = '/'
+            console.log(response)
```
Comment (olajohn-ajiboye) [inline src/pages/SignUpPage.js:55]:
```suggestion

```

File: src/pages/SignUpPage.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+const SignUpPage = () => {
+    // Form state
+    const [first_name, setFirsName] = useState('')
+    const [last_name, setLastName] = useState('')
+    const [email, setEmail] = useState('')
+    const [phone, setPhone] = useState('')
+    const [password, setPassword] = useState('')
+    const [password_confirmation, setPasswordConformation] = useState('')
+
+    //handel change of input for the form
```
Comment (olajohn-ajiboye) [inline src/pages/SignUpPage.js:14]:
```suggestion
    //handle change of input for the form
```

**Review by olajohn-ajiboye — CHANGES_REQUESTED** [review]

**Comment by boneswvb** [comment]
Hi Ola John,

 

I will delete all the stuff you requested.

 

I normally keep them for testing purposes until the component is working properly. I should be able to complete tonight and update to the repo.

**Pull request merged**